src/main/scala/mesosphere/marathon/state/MarathonStore.scala

```diff
@@ -113,7 +113,9 @@ class MarathonStore[S <: MarathonState[_, S]](
       Some(newState().mergeFromProto(bytes))
     }
     catch {
-      case e: InvalidProtocolBufferException => None
+      case e: InvalidProtocolBufferException =>
```

Comment:
Would it make sense to log the key for that state as well? I know it's not accessible from here, but maybe we should refactor this a bit.